### PR TITLE
Made script Python 2 + 3 compatible

### PR DESCRIPTION
**Pull request by aviaryan**
I saw there was a similar PR (#11) but that sacrificed Python 2 compatibility. This pull should allow both Python 2 and 3 to run the script without errors.

### CODE REVIEW

File: music_downloader.py [hunk 1]
```diff
@@ -3,19 +3,28 @@
...
-import urllib2
-from urllib import quote_plus as qp
+
+# Version compatiblity
+import sys
+isPython2 = True
+if (sys.version_info > (3,0)):
+	from urllib.request import urlopen
+	from urllib.parse import quote_plus as qp
+	isPython2 = False
```
Comment (yask123) [inline music_downloader.py]: better to  put raw_input=input here, this way we can avoid another comparison we are doing at line 21.

File: music_downloader.py [hunk 2]
```diff
@@ -3,19 +3,28 @@
...
+else:
+	from urllib2 import urlopen
+	from urllib import quote_plus as qp
 
 search = ''
 # We do not want to accept empty inputs :)
 while search == '':
-  search = raw_input('Enter songname/ lyrics/ artist.. or whatever\n> ')
+  search = raw_input('Enter songname/ lyrics/ artist.. or whatever\n> ') if isPython2 else input('Enter songname/ lyrics/ artist.. or whatever\n> ')
```
Comment (yask123) [inline music_downloader.py]: We wouldn't require this comparison then.

**Comment by yask123** [comment]
Thanks for helping. :+1: 
Can you re-push with the changes I mentioned above ?

**Comment by aviaryan** [comment]
Done. and btw, nice trick. :smiley_cat:

**Comment by yask123** [comment]
:100: :+1: 
LGTM
Thanks!

**Comment by dufferzafar** [comment]
@yask123 @aviaryan Keep up the good work!

**Comment by aviaryan** [comment]
@dufferzafar Thanks

**Pull request merged**